### PR TITLE
Add force credential reset option to globals

### PR DESCRIPTION
**Pull request by sjpadgett**
#### Short description of what this resolves:

Allow user admin to determine how portal credential force reset is handled
options are allow disallow or user selectable from dialog

### CODE REVIEW

**Comment by sjpadgett** [comment]
I put this up by itself so the portal login reset changes can be cherry picked to rel branch. Then the release branch version of global include will need to include the options in this PR.
Too many changes in master version to port back. Mainly logo options.

File: library/globals.inc.php [hunk 1]
```diff
@@ -3088,6 +3086,18 @@ function gblTimeZones()
...
         ),
 
+        'portal_force_credential_reset' => array(
+            xl('Portal Login Forced Credential Reset'),
+            array(
+                '0' => xl('Allow (Recommended)'),
+                '1' => xl('Disable'),
+                '2' => xl('User optional from credential dialog.')
+            ),
+            'enabled',
```
Comment (bradymiller) [inline library/globals.inc.php:3096]: above should be `0`

**Comment by bradymiller** [comment]
dance-bear

**Comment by sjpadgett** [comment]
Nice catch. That wouldn't broken anything it's still aggravating to do such
still silly mistakes.

**Comment by sjpadgett** [comment]
I'll modify release 7 production to add this then cherry pick the companion PR.

**Pull request merged**